src/Context/MainContext/SidebarManagementUtils.js

```diff
@@ -0,0 +1,51 @@
+export const addToVideoList = (videoList, itemToAdd) => {
+  if (videoList.length === 0) return [itemToAdd];
```

Comment:
This is redundant `videoList.some()` will return false if the list is empty and `line 5 ` will add the itemToAdd to `videoList`